src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -259,4 +261,84 @@ public void systemEndsPatientVisit() {
 		assertNull(visitsDashboardPage.getActiveVisit());
 		dashboardPage = visitsDashboardPage.goToPatientDashboard();
 	}
+
+	@When("a user clicks on Allergy2 link from Patient dashboard page2")
+	public void loadManageAllergy2() {
+		loadAllergiesPage();
+	}
+
+	@Then("the system loads Allergy2 board page")
+	public void loadsManageAllergy2() {
+		systemLoadsAllergiesPage();
+	}
+
+	@When("a user clicks Add New Allergy2 button")
+	public void addNewAllergy2() {
+		addKnownAllergy();
+	}
```

Comment:
You do not need to call it neither but as long as the step is captured in your scenario, it will look for that method automatically because the method is already.